mrmustard/lab_dev/circuit_components.py

```diff
@@ -136,6 +152,16 @@ def adjoint(self) -> CircuitComponent:
         ret._wires = ret.wires.adjoint()
         return ret
 
+    def dual(self) -> CircuitComponent:
```

Comment:
Please conjugate the representation as well for dual() and adjoint().

Reply:
We have the conj() function for representations. Just call then here~

Reply:
thanks! done